tests/fill-rule-evenodd-nano.svg

```diff
@@ -1,8 +1,7 @@
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="-10 -10 220 120">
   <defs/>
-  <path d="M50,0 21,90 98,35 2,35 79,90 z"/>
+  <path d="M50,0 L21,90 L98,35 L2,35 L79,90 Z"/>
```

Comment:
so, not only picosvg paths will be absolute, but repeated implicit commands will also be "exploded". This is because iterating over SVGPath always yields exploded commands, and we are now applying `absolute()` to all shapes.
I tried to avoid exactly that when I added round_floats method to SVGPath, by iterating with parse_svg_path(self.d, exploded=False).
Are we ok with this change?

Reply:
IMHO it's a good change. picosvg is supposed to be simple, explicit absolute commands is just about as simple as you can get. I'm resolving but please reopen if you disagree.

Reply:
sgtm